### PR TITLE
Gettable datetime filter

### PR DESCRIPTION
**Pull request by cameroncaci**
[Epic](https://www13.v1host.com/USTRANSCOM38/Epic.mvc/Summary?oidToken=Epic%3a810602)
* Increase version to 0.4.1.0
* Add conditional check for date time value ranges provided
  * Added TwoValueDateTimeFilter usage should these be provided to request specific date ranges of data from TRDM
* Added proper system settings if trust store and keypair store is found to already be existing
  
 Example output:
```
<soapenv:Envelope xmlns:ret="http://trdm/ReturnTableService" xmlns:soapenv="http://www.w3.org/2003/05/soap-envelope">
   <soapenv:Header/>
   <soapenv:Body>
      <ret:getTableRequestElement>
         <ret:input>
            <ret:TRDM>
               <ret:physicalName>LN_OF_ACCT</ret:physicalName>
               <ret:returnContent>true</ret:returnContent>
               <ret:columnFilters>
                  <ret:columnFilter>
                     <ret:column>LAST_UDP_DT</ret:column>
                     <ret:columnFilterTypes>
                        <ret:twoValueDateTimeFilter>
                           <ret:filterType>IS_BETWEEN</ret:filterType>
                           <ret:filterValue>2023-08-01T00:00:00.000Z</ret:filterValue>
                           <ret:filterValue>2023-00-31T23:59:59.999Z</ret:filterValue>
                        </ret:twoValueDateTimeFilter>
                     </ret:columnFilterTypes>
                  </ret:columnFilter>
               </ret:columnFilters>
            </ret:TRDM>
         </ret:input>
      </ret:getTableRequestElement>
   </soapenv:Body>
</soapenv:Envelope>
```

The following has not been implemented yet:
```
<ret:returnRowStatus></ret:returnRowStatus>
<ret:returnLastUpdate></ret:returnLastUpdate>
```

### CODE REVIEW

**Comment by cameroncaci** [comment]
Added example to description

File: src/main/java/com/milmove/trdmlambda/milmove/service/GetTableService.java [hunk 1]
```diff
@@ -96,6 +107,30 @@ private GetTableResponse createSoapRequest(GetTableRequest request)
         trdm.setContentUpdatedSinceDateTime(DatatypeFactory.newInstance()
                 .newXMLGregorianCalendar(request.getContentUpdatedSinceDateTime()));
 
+        // Check if the optional fields of date time filters were provided
```
Comment (r-mettler) [inline src/main/java/com/milmove/trdmlambda/milmove/service/GetTableService.java:110]: Looks good to me. I can't see any obvious issues with it. Go test in production and find out I guess ...eek 😬

**Review by traskowskycaci — COMMENTED** [review]
My java is rusty but this looks fine.

**Comment by danieljordan-caci** [comment]
You forgot to tag me but HERE IS MY COMMENT ANYWAY

**Pull request merged**